cpp/src/tests/sssp/.clang-format

```diff
@@ -0,0 +1,12 @@
+---
+# This configuration requires clang-format 3.8 or higher.
+BasedOnStyle: chromium 
+AlignOperands: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakAfterDefinitionReturnType: None
+ColumnLimit: 79
+BinPackArguments: false
+BinPackParameters: false
+Standard: Cpp11
+...
+
```

Comment:
cuGraph currently does not use clang-format, so this is out of sync with the cuGraph repository.

That being said, cuDF and cuML are incorporating clang-format(
https://github.com/rapidsai/cudf/issues/540
https://github.com/rapidsai/cuml/pull/616
)

We should consider incorporating this as well at the cuGraph level (@BradReesWork, @afender).

Reply:
Yes, that caught my attention too. 
Let's add it @pgera. Just make sure it is consistent with the rest of RAPIDS.

Reply:
https://github.com/rapidsai/cugraph/issues/326

Reply:
@pgera and just FYI, RAPIDS is officially using C++14.